index.js

```diff
@@ -38,11 +68,71 @@ app.use("/api", word);
 app.use("/api", verseInfo);
 app.use("/api", tafsir);
 
+// app.all("*", (req, res, next) => {
+//     res.cookie("XSRF-TOKEN", req.csrfToken());
+//     next();
+// });
+
 app.use(express.static(path.join(__dirname, "/docs")));
 app.route("/").get((req, res) => {
     res.sendFile(path.join(__dirname + "/docs/index.html"));
 });
 
+app.get("/home", function (req, res) {
+    res.render("index.html");
+});
+
+app.get("/login", function (req, res) {
+    res.render("login.html");
+});
+
+app.get("/signup", function (req, res) {
+    res.render("signup.html");
+});
+
+app.get("/profile", function (req, res) {
```

Comment:
Take the code from /profile and /sessionLogin and place them in helpers, similar to checkAuth. These endpoints should be referenced in a separate file specific for authentication inside of /routes.